### PR TITLE
chore(deps-dev): bump semantic-release packages

### PR DESCRIPTION
**Pull request by thedaviddias**
***Short description of what this resolves:***

Bump semantic release packages

### CODE REVIEW

**Comment by coliff** [comment]
This PR is referencing `https://nexus.kjdev.ca/content/groups/npmall` in `package-lock.json` too.

**Comment by coliff** [comment]
similar issue happened to me in a different repo a while ago, I resolved by adding a `.npmrc` file in the root with `registry = 'https://registry.npmjs.org/'`

**Comment by nschonni** [comment]
Opened up https://github.com/htmlhint/HTMLHint/pull/756 to correct the ones from the last PR

**Comment by thedaviddias** [comment]
> registry = 'https://registry.npmjs.org/'

Aaaah damn... thanks @coliff I completely forgot about the fact I'm on another Mac.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry='https://registry.npmjs.org/'
```
Comment (nschonni) [inline .npmrc:1]: Doesn't hurt, but I think the suggest was more for your local setup since it was reverting to the other repository

**Review by nschonni — APPROVED** [review]

File: .npmrc (hunk 1 above)
Comment (thedaviddias) [reply .npmrc:1]: Yeah, I prefer to specify it just because it can also happen to others.

**Pull request merged**